### DISCUSSION TITLE
Modelling internal wave in continuously stratified fluid over topography

### DISCUSSION THREAD

**Discussion by jianzhou722** [xcompact3d/Incompact3d#210] [General]
Hi everyone,

I am trying to model the generation and propagation of internal wave in continuously stratified flow over bottom topography. The figure below shows an example where isopycnals are disturbed by the bottom obstacle. Would Incompact3d be able to simulate this kind of problem? Thanks!

![Screenshot 2023-07-17 234311](https://github.com/xcompact3d/Incompact3d/assets/36316186/1dd6a86b-6b82-4a6e-b2d0-d18c36f9e07c)

**Comment by slaizet** [comment]
Hi,
Yes it should be possible to simulate these internal waves.
Best,
Sylvain
